modules/tinymce/src/plugins/table/main/ts/Plugin.ts

```diff
@@ -18,18 +19,19 @@ import { Clipboard as FakeClipboard } from './core/Clipboard';
 import * as TableFormats from './core/TableFormats';
 import * as TabContext from './queries/TabContext';
 import CellSelection from './selection/CellSelection';
-import * as Ephemera from './selection/Ephemera';
-import { Selections } from './selection/Selections';
+import { ephemera } from './selection/Ephemera';
```

Comment:
It looks weird to me that this is lower case - I assume I messed up the export?

Reply:
That's what you exported the variable name as, so that's expected. You'd have to make it a capital in the variable if you wanted it to be that here as well. I'm pretty sure you do want it a capital, so that the types from darwin merge?